src/main/webapp/calendar-script.js

```diff
@@ -43,8 +66,95 @@ function getEvents(calendar) {
         title: event.name,
         start: event.strStartTime,
         end: event.strEndTime,
-        allDay: false
+        allDay: false,
+        extendedProps: {
+          stringDate: event.strStartTime,
+          address: event.address,
+          placeId: event.placeId, 
+          openTime: '9AM',
+          closeTime: '5PM',
+        }
       });
     });
   });
 }
+
+function createMap(modalBody, eventObj) {
+  const infoDisplay = document.createElement('div');
+  let address = document.createElement('p');
+  var urlFormatAddress = eventObj.extendedProps.address.replace(/\s/g, '+')
+  var url = 'https://www.google.com/maps/search/?api=1&query=' + urlFormatAddress + '&query_place_id=' + eventObj.extendedProps.placeId;
+  infoDisplay.innerHTML = '<b>Address: </b><a href=' + url + '>' + eventObj.extendedProps.address + '</a>' + '<br>';
+  infoDisplay.appendChild(address);
+  modalBody.appendChild(infoDisplay)
+
+  // create new div to hold map
+  const mapDis = document.createElement('div');
+  mapDis.id = 'map';
+  modalBody.appendChild(mapDis);
+
+  // instantiate map
+  map = new google.maps.Map(document.getElementById('map'), {
+      zoom: 13,
+      center: new google.maps.LatLng(0, 0)
```

Comment:
Do you have to set a value for the `center` attribute initially? I removed it on the local test and nothing changed, so I think you can remove it.